licenses.yaml

```diff
@@ -364,7 +364,7 @@ name: Guava
 license_category: binary
 module: java-core
 license_name: Apache License version 2.0
-version: 31.1-jre
+version: 32.0.1-jre
```

Comment:
any reason we can't move to 32.1.3? I'm fine if we stick to 32.0.1 if there are more invasive changes required to do the full jump, just trying to understand.

Reply:
I ran into issues with integration tests with the higher version and instead of taking time to investigate it further I went with the first CVE free available version that tested OK.

Reply:
that's fine, thanks for giving it a shot